lib/i18n/hygiene/wrapper.rb

```diff
@@ -8,7 +8,8 @@ module Hygiene
     # queryable.
     class Wrapper
 
-      def initialize(keys_to_skip:)
+      def initialize(keys_to_skip: [], locales: ::I18n.available_locales)
```

Comment:
Can we trust `::I18n.available_locales` to always return something sensible?

Is it always set? I think we only started setting it in our apps recently, and before that it *may* have been an empty array?